framework/src/problems/FEProblemBase.C

```diff
@@ -4382,6 +4383,28 @@ FEProblemBase::backupMultiApps(ExecFlagType type)
   }
 }
 
+void
+FEProblemBase::reinitMultiApps(ExecFlagType type, std::string type_name)
+{
+  const auto & multi_apps = _multi_apps[type].getActiveObjects();
+
+  if (multi_apps.size())
+  {
+    _console << COLOR_CYAN << "\nReinitializing " << type_name << "s on " << type.name()
+             << COLOR_DEFAULT << std::endl;
+
+    for (const auto & app : multi_apps)
+      if (app->type() == type_name)
+        app->reinitialize();
```

Comment:
no need for a print if the user did not request a re-initialization

Reply:
Got it, restructured the corresponding part.